sema/subyt/sources.py

```
@@ -315,7 +409,7 @@ class JsonFileSource(Source):
         Source producing iterator over data-set coming from json on file
         """
 
-        def __init__(self, json_file_path: Path) -> None:
+        def __init__(self, json_file_path: Path, config: dict = {}) -> None:
```

Ruff (0.8.2)

Unused method argument: `config`

(ARG002)